cpp/oneapi/dal/backend/primitives/stat/cov_dpc.cpp

```diff
@@ -66,7 +67,10 @@ inline sycl::event compute_covariance(sycl::queue& q,
     const std::int64_t n = row_count;
     const std::int64_t p = sums.get_count();
     const Float inv_n = Float(1.0 / double(n));
-    const Float inv_n1 = (n > Float(1)) ? Float(1.0 / double(n - 1)) : Float(1);
+    Float multiplier = (n > 1) ? Float(1.0 / double(n - 1)) : Float(1);
+    if (bias) {
+        multiplier = inv_n;
+    }
```

Comment:
```suggestion
    const Float inv_n1 = (n > Float(1)) ? Float(1.0 / double(n - 1)) : Float(1);
    const Float multiplier = bias ? inv_n1 : inv_n;
```
Please don't remove constancy from `multiplier`.